CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+Changes by Version
+==================
```

Comment:
Could be used the same formatting symbols as in other markdown files in this repository. e.g. `#` for headers, `*` for lists

Reply:
I was trying to be uniform by copying the style in the go CHANGELOG ;) : https://raw.githubusercontent.com/opentracing/opentracing-go/master/CHANGELOG.md\

We also use `.rst` instead of `.md` in some repos. 🤷‍♂️ 

I prefer `#` and `*` so I'll switch to that. Maybe we can standardize and change the others as part of the upcoming "language maintainer" initiative.